### ISSUE TITLE
[issue name] version 0.1.66

### ISSUE THREAD

**Issue by junjunwang2016** [iron-fish/ironfish#3368]
### What happened?

when i update 0.1.65 to 0.1.66:
Your database needs to be upgraded (v17 vs v19).
Run "ironfish migrations:start" or "ironfish start --upgrade"

then i run this code ironfish start --upgrade, but new error:
Applying 2 migrations:
  018-backfill-wallet-assets
  019-backfill-wallet-assets-from-chain

Running 018-backfill-wallet-assets...
Backfilling assets for 1 accounts

  Backfilling assets for account default
Error applying 018-backfill-wallet-assets
Error: Supply should be non-null for asset
    at Assert.isNotNull (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\assert.js:23:19)
    at Account.saveMintsToAssetsStore (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\wallet\account.js:176:33)
    at async Migration018.forward (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\migrations\data\018-backfill-wallet-assets.js:27:17)
    at async Migrator.migrate (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\migrations\migrator.js:118:17)
    at async IronfishNode.openDB (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\node.js:189:13)
    at async Object.waitForOpen (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\utils\node.js:16:13)
    at async Start.start (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\build\src\commands\start.js:105:9)
    at async Start.run (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\build\src\command.js:24:13)
    at async Start._run (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@oclif\core\lib\command.js:80:22)
    at async Config.runCommand (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@oclif\core\lib\config\config.js:272:25)

### Version

v0.1.66

### Debug output

```shell
PS C:\Users\Administrator> ironfish debug
Database cannot be opened, skipping output that requires database.

DatabaseVersionError: Your database needs to be upgraded (v17 vs v19).
Run "ironfish migrations:start" or "ironfish start --upgrade"

    at LevelupDatabase.upgrade (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\storage\levelup\database.js:83:19)
    at async WalletDB.open (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\wallet\walletdb\walletdb.js:105:9)
    at async Wallet.open (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\wallet\wallet.js:124:9)
    at async IronfishNode.openDB (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@ironfish\sdk\build\src\node.js:193:13)
    at async Debug.start (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\build\src\commands\debug.js:21:13)
    at async Debug.run (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\build\src\command.js:24:13)
    at async Debug._run (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@oclif\core\lib\command.js:80:22)
    at async Config.runCommand (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@oclif\core\lib\config\config.js:272:25)
    at async Object.run (C:\Users\Administrator\AppData\Roaming\npm\node_modules\ironfish\node_modules\@oclif\core\lib\main.js:74:5)

Iron Fish version               0.1.66 @ 12abe5b
Iron Fish library               0.0.43 @ 12abe5b
Operating system                Windows_NT x64
CPU model(s)                    Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
CPU threads                     6
RAM total                       15.91 GiB
Heap total                      4.00 GiB
Node version                    v18.14.0
ironfish in PATH                true
Garbage Collector Exposed       false
Telemetry enabled               true
Node name                       junjun
Block graffiti                  junjun
```

### Graffiti

junjun

**Comment by thoreausolo** [comment]
same process same result

**Comment by hughy** [comment]
Resolved by #3380
